.github/workflows/ci.yaml

```diff
@@ -151,6 +151,9 @@ jobs:
     name: SQL Logic Tests
     runs-on: ubuntu-latest-8-cores
     needs: ["build"]
+    strategy:
+      matrix:
+        protocol: ['postgres','flightsql']
```

Comment:
Nice, looking forward to adding the rpc tests too.

Reply:
sorry for the weird flight merge.